comment.go

```diff
@@ -119,17 +126,24 @@ func takeLastComment(list []Visitee) (*Comment, []Visitee) {
 // mergeOrReturnComment creates a new comment and tries to merge it with the last element (if is a comment and is on the next line).
 func mergeOrReturnComment(elements []Visitee, lit string, pos scanner.Position) *Comment {
 	com := newComment(pos, lit)
-	// last element must be a comment to merge +
-	// do not merge c-style comments +
-	// last comment line was on previous line
-	if esize := len(elements); esize > 0 {
-		if last, ok := elements[esize-1].(*Comment); ok &&
-			!last.Cstyle &&
-			pos.Line <= last.Position.Line+len(last.Lines) { // less than because last line of file could be inline comment
-			last.Merge(com)
-			// mark as merged
-			com = nil
-		}
+	esize := len(elements)
+	if esize == 0 {
+		return com
+	}
+	// last element must be a comment to merge
+	last, ok := elements[esize-1].(*Comment)
+	if !ok {
+		return com
+	}
+	// do not merge c-style comments
```

Comment:
Why? c-style comments can be valid.
  
```proto
int64 hello = 1; /* inline comment */
int64 foo = 2; /* inline
  multiline
  comment */
```

Reply:
Would you want this 
```
/* 
one
two
*/
/*
three
four
*/
```
to be merged into 
/*
one
two
three
four
*/
```
? Clearly, c-style comments already allow multiline comments so I don't want to ignore the intention of the writer.
The scanner already reads all lines of the c-style comment.  I added an additional test to verify.

Reply:
as c-style Comment values already have multiple lines, merging 2 Comment values is not what we want.